### ISSUE TITLE
Stop Button

### ISSUE THREAD

**Issue by Stephan1908** [Deejayfool/hass-shutter-card#46]
Cool Card, thank you very much.
Is it possible to hide the stop button?

**Comment by dermorpher** [comment]
do you have an solution found?

**Comment by Stephan1908** [comment]
No

**Comment by michapr** [comment]
+1 from me for hide the stop button

**Comment by dannyb86** [comment]
yep, would be nice. because homekit does not support stop on my somy io shutters. then pushing de stop button generates irritating error messages for the family members. :-)

**Comment by scarey** [comment]
My PR lets you disable the stop button.  Not as good as hiding but would help with your issue.
